docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,153 @@
+.. _acsincome-data:
+Retiring Adult: ACSIncome
+-------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The ACSIncome dataset is one of five datasets created by Ding et al. [1]_ 
+as an improved alternative to the popular UCI Adult dataset.
+The authors compiled data from the American Community Survey (ACS) Public Use Microdata Sample (PUMS). 
+Note that this is a different source than the Annual Social and Economic Supplement (ASEC) 
+of the Current Population Survey (CPS) used to construct the original UCI Adult dataset.
+Ding et al. [1]_ filtered the data such that ACSIncome only includes individuals above 16 years old 
+who worked at least 1 hour per week in the past year and had an income of at least $100.
```

Comment:
A few words about why they filtered it that way could be helpful

Reply:
I'm actually unsure why these filters were chosen. For example, page 7 of the paper simply states

```
ACSIncome: predict whether an individual’s income is above $50,000, after filtering the ACS PUMS data sample to only include individuals above the age of 16, who reported usual working hours of at least 1 hour per week in the past year, and an income of at least $100.
```


Page 21 in the SI restates the filters, but again does not offer an explanation. In contrast, the ACSPublicCoverage is filtered to individuals under the age of 65, and the authors explicitly state that the purpose was to focus on people who are not eligible for Medicare.